docs/deploy.md

```diff
@@ -0,0 +1,87 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# Deploy manually
+
+## Clone the project
+
+```sh
+$ git clone https://github.com/apache/apisix-dashboard.git
+
+$ cd apisix-dashboard
+```
+
+## Build the manager-api
+
+The `manager-api` is used to provide APIs for Dashboard, just like a bridge between the Apache APISIX and the Dashboard. Here are the steps to build it manually:
+
+1. We need `Go` 1.13+ and `Lua` 5.1+ to be preinstalled.
+
+2. Check environment variables
+
+- enable Go MODULE
+
+```sh
+$ go env -w GO111MODULE=on
+```
+
+- According to your local deployment environment, check the environment variables in `./api/run.sh`, modify the environment variables if needed. For example, change the ETCD endpoints to your ETCD instances work with APISIX:
+
+```
+export APIX_ETCD_ENDPOINTS="127.0.0.1:2379"
+```
+
+If you have multiple instances, please use commas to separate:
+
+```
+export APIX_ETCD_ENDPOINTS="127.0.0.1:2379,127.0.0.1:3379"
+```
+
+- For most users in China, we could use [Goproxy](https://goproxy.cn/) to speed up downloading modules.
+
+```sh
+$ go env -w GOPROXY=https://goproxy.cn,direct
+```
+
+3. Build and Run
+
+```sh
+$ ./api/run.sh &
+```
+
+## Build the Dashboard
```

Comment:
`front-end` instead of `Dashboard` would be better.I think `Dashboard` is contain front-end and manager api,How do you think?

Reply:
LGTM